packages/sdk/src/evm/core/classes/drop-claim-conditions.ts

```diff
@@ -747,10 +766,11 @@ export class DropClaimConditions<
     const addressToClaim = address
       ? address
       : await this.contractWrapper.getSignerAddress();
+    const activeCondition = (await this.getActive({ returnInactive: true })) as ClaimCondition; 
```

Comment:
Should `returnInactive` here be `true`? If not, how we should fail this request, if there isn't an active claim condition?